### DISCUSSION TITLE
Send full item in webhook

### DISCUSSION THREAD

**Discussion by hmirzoyan-chwy** [directus/directus#5243] [Feature Requests]
Version: 60 latest

When I trigger a WebHook it only sends the ID's & the schema .. but not the fields and field values themselves

**Comment by rijkvanzanten** [comment]
This is correct. The webhook passes on the hook's payload and data, which doesn't include the full item. I've transferred your issue to a feature request 👍🏻

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
